[2022-04-24 23:49:01,266] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: local_2_gcs_2_bq_dag.crash_local_to_gcs_task manual__2022-04-24T22:27:24.245486+00:00 [queued]>
[2022-04-24 23:49:01,280] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: local_2_gcs_2_bq_dag.crash_local_to_gcs_task manual__2022-04-24T22:27:24.245486+00:00 [queued]>
[2022-04-24 23:49:01,280] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-24 23:49:01,281] {taskinstance.py:1239} INFO - Starting attempt 8 of 8
[2022-04-24 23:49:01,282] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-24 23:49:01,298] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): crash_local_to_gcs_task> on 2022-04-24 22:27:24.245486+00:00
[2022-04-24 23:49:01,305] {standard_task_runner.py:52} INFO - Started process 3976 to run task
[2022-04-24 23:49:01,313] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'local_2_gcs_2_bq_dag', 'crash_local_to_gcs_task', 'manual__2022-04-24T22:27:24.245486+00:00', '--job-id', '159', '--raw', '--subdir', 'DAGS_FOLDER/download_2_gcs_dag_V3.py', '--cfg-path', '/tmp/tmpicte48rh', '--error-file', '/tmp/tmp2p9jjre0']
[2022-04-24 23:49:01,315] {standard_task_runner.py:77} INFO - Job 159: Subtask crash_local_to_gcs_task
[2022-04-24 23:49:01,424] {logging_mixin.py:109} INFO - Running <TaskInstance: local_2_gcs_2_bq_dag.crash_local_to_gcs_task manual__2022-04-24T22:27:24.245486+00:00 [running]> on host 2bdc5459fc64
[2022-04-24 23:49:01,509] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=local_2_gcs_2_bq_dag
AIRFLOW_CTX_TASK_ID=crash_local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-24T22:27:24.245486+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-24T22:27:24.245486+00:00
[2022-04-24 23:49:06,510] {python.py:175} INFO - Done. Returned value was: None
[2022-04-24 23:49:06,527] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=local_2_gcs_2_bq_dag, task_id=crash_local_to_gcs_task, execution_date=20220424T222724, start_date=20220424T234901, end_date=20220424T234906
[2022-04-24 23:49:06,608] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-24 23:49:06,657] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
